src/OpenTelemetry.Abstractions/Trace/Export/SpanData.cs

```diff
@@ -118,25 +112,24 @@ public sealed class SpanData : ISpanData
                         Timestamp startTimestamp,
                         IAttributes attributes,
                         ITimedEvents<IEvent> events,
-                        ITimedEvents<IMessageEvent> messageEvents,
                         ILinks links,
                         int? childSpanCount,
                         Status status,
                         SpanKind kind,
                         Timestamp endTimestamp)
         {
-            if (messageEvents == null)
+            if (events == null)
             {
-                messageEvents = TimedEvents<IMessageEvent>.Create(new List<ITimedEvent<IMessageEvent>>(), 0);
+                events = TimedEvents<IEvent>.Create(new List<ITimedEvent<IEvent>>(), 0);
             }
 
-            var messageEventsList = new List<ITimedEvent<IMessageEvent>>();
-            foreach (ITimedEvent<IMessageEvent> timedEvent in messageEvents.Events)
+            var eventsList = new List<ITimedEvent<IEvent>>();
```

Comment:
it doesn't seem like eventsList or timedMessageEvents are used anywhere. perhaps there was some bug here?